tsconfig.json

```diff
@@ -1,51 +1,51 @@
 {
-  "compilerOptions": {
-    "baseUrl": "./packages",
-    "paths": {
-      "devextreme-angular": [
-        "devextreme-angular/npm/dist"
-      ],
-      "devextreme-angular/*": [
-        "devextreme-angular/npm/dist/*"
-      ],
-      "devextreme-react": [
-        "devextreme-react/npm"
-      ],
-      "devextreme-react/*": [
-        "devextreme-react/npm/*"
-      ],
-      "devextreme-vue": [
-        "devextreme-vue/src"
-      ],
-      "devextreme-vue/*": [
-        "devextreme-vue/src/*"
-      ],
-      "devextreme-monorepo-tools": [
-        "devextreme-monorepo-tools/src"
+    "compilerOptions": {
```

Comment:
Are there meaningful changes in this file?

Reply:
No
Reverted